examples/libsodium/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.0)
+project(libsodium-example)
+
+set(HUNTER_CONFIGURATION_TYPES Release) #Release or Debug
```

Comment:
Why not, this is testing. Also cache of this package will be uploaded according to how example is build. so effectively you making binary cache useless for default configuration